package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "assert-http",
-  "version": "1.0.3",
+  "version": "1.1.3",
```

Comment:
You can leave the version bump out of the PR -- version bumps are best done from `master` since another PR may also make its way into the next version.